README.md

```diff
@@ -28,28 +36,30 @@ To install the Biome IntelliJ Plugin, Head over to [official plugin page](https:
 2. Press `⌘Сmd,` to open the IDE settings and then select Plugins.
```

Comment:
I believe this could be improved with a `kbd` tag

Reply:
Right. I've incorporated the <kbd>kbd</kbd> tag to enhance the representation of the command:

Press <kbd>⌘Cmd</kbd>, to open the IDE settings and then select Plugins.